docs/fides/docs/development/documentation.md

```diff
@@ -14,11 +14,11 @@ To write concept docs, add Markdown files to the `docs/fides/docs/` directory (o
 
 ### Capitalization
 
-Concepts that refer to proper nouns or are trademarked should always be capitalized, including "Fides" and "FidesAPI".
+Concepts that refer to proper nouns or are trademarked should always be capitalized. The exception here is fidesctl and fidesops, which are lowercased as a stylistic choice.
```

Comment:
@PSalant726 is this in alignment with what you were going off of? as in does this policy now match what the direction from marketing is?

Reply:
Yes. We could add fideslang to the list, but I don't think we need to.